ui/tests/UHC-1.test.ts

```diff
@@ -0,0 +1,60 @@
+import { test, expect } from '@Test';
+
+test.describe('UHC-1 (test ID)', () => {
+    test('Registration new customer with valid data and checking user data reset after logout (test title)', async ({ browser, page, baseURL  }) => {
+        await page.goto(baseURL as string, {waitUntil: 'domcontentloaded'});
+
+        const hoverMyAccount = page.locator('//div[text()="My Account"]');
+        await hoverMyAccount.hover({noWaitAfter: true});
+
+        const loginButton = page.locator('//ul//li//button[text()="Log in"]');
+        await loginButton.click();
+
+        const registrationButton = page.locator('//div//button//span//span[text()=\'Create UHCGlasses.com Account\']');
+        await registrationButton.click();
+
+        const firstName = page.getByLabel('First Name');
+        await firstName.type('Ivan', { delay: 100 });
+
+        const lastName = page.getByLabel('Last Name');
+        await lastName.type('Ivanov', { delay: 100 });
+
+        const email = page.getByLabel('Email')
+        await email.type('test@test.com', { delay: 100 });
```

Comment:
The test will fail on the second run.
It is impossible to register users with the same (static) email.